### ISSUE TITLE
Slack - API Bot User

### ISSUE THREAD

**Issue by vlnrajesh** [nisabek/icinga2-slack-notifications#20]
Can we user Slack API bot user with this plugin instead of Web hook URL?

**Comment by staenker** [comment]
Yes, we can. The when is a question though. The web hook is working quite and fits into the simplicity approach if this plugin very well. One of the main reasons I starting writing this is, that icinga ships with everything that is required, no bash scripts, no dependencies. So it kinda does the job. You could argue, that an Authorization header token might be more secure than a web hook URL, but as long as you use https and keep the web hook  safe, you are safe. Given that, this feature would not be high on our priority list and it might take an undefined amount time > 6 months; <= infinity to get implemented. You can of course change that time frame by gettin your hands dirty and implement the feature request in a way, that also allows users to continue using web hooks.
Cheers,
Richard
